pages/admin/courses/course_list.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div id="page">
+  <v-container class="layout" v-if="!flagView"></v-container>
+  <div v-else>
+      <header class="title">
+        <h1>MEUS CURSOS</h1>
+        <v-btn class="mx-2 btn-icon" icon>
+          <v-icon dark id="plus-icon">mdi-plus-circle</v-icon>
+        </v-btn>
+      </header>
+      <div class="body-list">
+        <v-card v-for="item in list" v-bind:key="item.id" class="v-card-border">
+          <div class="content">
+            <div class="img-mask">
+              <img :src="item.thumbUrl" alt />
+            </div>
+            <div class="info-text">
+              <h1>{{item.title}}</h1>
+              <p>{{item.description}}</p>
+            </div>
+          </div>
+          <div class="group-buttons">
+            <v-btn class="btn-item bg-blue">
+              <v-icon class="text-white">mdi-border-color</v-icon>
+            </v-btn>
+            <v-btn class="btn-item bg-danger" @click="deleteCourse(item.id)">
+              <v-icon class="text-white">mdi-delete</v-icon>
+            </v-btn>
+          </div>
+        </v-card>
+      </div>
+      <navigation-bar />
+    </div>
+  </div>
+</template>
+<router>
+{
+    path: '/admin/listar-cursos'
+  }
+</router>
+<script>
+import NavigationBar from '~/components/NavigationBar.vue'
+import http from '~/services/http/generic'
+export default {
+  data: () => ({
+    list: [],
+    flagView : false
+  }),
+  components: {
+    NavigationBar,
+  },
+  methods: {
+    getAllCourses() {
+      http
+        .getAll("/api/v1/course")
+        .then(res => {
+          list = res.data
+          this.flagView = (this.list.length > 0)  
+        })
+        .catch(err => {
+          alert(err)
+        })
+    },
+    deleteCourse(id){
+      http.delete(`/api/v1/course/${id}`).then(
+        res=>{
+          alert("Curso excluído com suceso!")
+        }
+      ).catch((err)=>{
+        console.error(err)
+        alert("Erro ao excluir o curso!")
+      })
+    }
+  },
+  mounted() {
+    this.getAllCourses()
```

Comment:
```suggestion
    this.loadAllCourses()
```
Continuação da sugestão de nome do método